index.js

```diff
@@ -100,6 +100,7 @@ async function run() {
     });
     core.debug(`Created Pull Request #${pr.number} in ${owner}/${repo}`);
     core.debug(`Pull Request link: ${pr.html_url}`);
+    core.setOutput('pr_url', pr.html_url);
```

Comment:
@barraponto Além de deixar disponível a url aqui é necessário em mais dois lugares:

- README.md (https://github.com/actions/javascript-action exemplo)
- action.yml (https://github.com/actions/javascript-action/blob/main/action.yml)

Reply:
Na verdade vc nem precisa exportar a variável, como seu objetivo é comentar no PR, vc pode fazer:

```javascript
await client.rest.issues.createComment({
  owner,
  repo,
  issue_number, // faz apenas coletar esse parâmetro
  body: `betrybe/trybe PR available at ${pr.html_url}`
});
```

Reply:
https://octokit.github.io/rest.js/v18#issues-create-comment

Reply:
realmente, usar o cliente de github deve ser mais rápido inclusive. não seria o mesmo que #11 ?
se essa for a melhor opção, prefiro mergear aquela outra issue.

eu tinha escolhido uma step separada pra minimizar minahs alterações aqui.
a action que eu escolhi inclusive evita comentários duplicados.

Reply:
@jeanpsv @barraponto Pessoal, como prosseguimos aqui?